lib/reporting/identity_verification_report.rb

```diff
@@ -17,15 +17,27 @@ class IdentityVerificationReport
     attr_reader :issuer, :time_range
 
     module Events
+      IDV_DOC_AUTH_WELCOME = 'IdV: doc auth welcome visited'
+      IDV_DOC_AUTH_GETTING_STARTED = 'IdV: doc auth getting_started visited'
       IDV_DOC_AUTH_IMAGE_UPLOAD = 'IdV: doc auth image upload vendor submitted'
-      IDV_GPO_ADDRESS_LETTER_REQUESTED = 'IdV: USPS address letter requested'
-      USPS_IPP_ENROLLMENT_CREATED = 'USPS IPPaaS enrollment created'
       IDV_FINAL_RESOLUTION = 'IdV: final resolution'
       GPO_VERIFICATION_SUBMITTED = 'IdV: GPO verification submitted'
       USPS_ENROLLMENT_STATUS_UPDATED = 'GetUspsProofingResultsJob: Enrollment status updated'
 
-      def self.all_events
-        constants.map { |c| const_get(c) }
+      IDV_FINAL_RESOLUTION_VERIFIED = 'IdV: final resolution - Verified'
+      IDV_FINAL_RESOLUTION_FRAUD_REVIEW = 'IdV: final resolution - Fraud Review Pending'
+      IDV_FINAL_RESOLUTION_GPO = 'IdV: final resolution - GPO Pending'
+      IDV_FINAL_RESOLUTION_IN_PERSON = 'IdV: final resolution - In Person Proofing'
```

Comment:
these are titles, not event names right? What if we made these symbols instead of constants, so that we could stil have the `all_events` method?

so we'd do like

```ruby
event_users[:idv_final_resolution_verified] << 'blah'
```
and that way we're not mixing string events with non-events? it doesn't looks like we're using the names of these in the report output anyways

Reply:
or another idea: maybe we make a second module like `module Results` for these new ones, so that the events constants are all event names

Reply:
I like the `Results` more.

The thing that gets messy with symbols is `IdentityVerificationReport#data`. Then you have a mix of symbols and strings are you keys in the resulting hash. Idk if that is intended to have callers but it is public and it does have tests to make sure it is shaped the way we expect.

Reply:
🤷 I don't feel so strongly about that, we could put them in different hashes or change the tests but ya I liked that the `Events` module had all real event names, and the `.all_events` did what it said on the tin